client/components/AddNew.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react'
+import { Form } from 'semantic-ui-react'
+import { Link } from 'react-router-dom'
+
+export default class AddNew extends Component {
+  state = {}
```

Comment:
Don't forget to include the default form values in here, like:
```js
state = {
  username: '',
  threat: ''
}
```
or whatever.

Reply:
Added username and threat to state.